packages/edit-site/src/components/sidebar-button/style.scss

```diff
@@ -14,7 +14,7 @@
 		outline: 3px solid transparent;
 	}
 
-	&:hover,
+	&:hover:not(:disabled,[aria-disabled=true]),
 	&:focus-visible,
 	&:focus,
 	&:not([aria-disabled="true"]):active,
```

Comment:
Should we add `:disabled` here too, to align with the selector introduced above?

Reply:
Also, looking forward to color theming which will hopefully make such overrides unneded